src/auth/ha-auth-flow.ts

```diff
@@ -79,13 +76,17 @@ export class HaAuthFlow extends LitElement {
 
   protected render() {
     return html`
+      <style>
+        .action {
+          margin: 24px 0 8px;
+          text-align: center;
+        }
+        .store-token {
+          margin-top: 10px;
+          margin-left: -16px;
+        }
+      </style>
```

Comment:
Let's keep this in `static get styles`

Reply:
I thought Lit would do that for use, but ok 🤷‍♂️

Reply:
Because that's something that's done in the default `createRenderRoot` implementation.